src/components/NewEventDialog.js

```diff
@@ -0,0 +1,158 @@
+import React, { useCallback } from 'react';
+import { useMutation } from '@apollo/react-hooks';
+import useForm from 'react-hook-form';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { TextField, Button, Grid, Box } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import useStyle from './UseStyle';
+
+const { DateTime } = require('luxon');
+
+const mutation = gql`
+  mutation addEvent(
+    $title: String!
+    $start: String!
+    $end: String!
+    $description: String!
+  ) {
+    addEvent(
+      title: $title
+      start: $start
+      end: $end
+      description: $description
+    ) {
+      id
+      title
+      start
+      end
+      description
+    }
+  }
+`;
+
+function NewEventDialog(props) {
+  const { onClose, open, selectedDate } = props;
+  const classes = useStyle();
+
+  const { handleSubmit, register, errors } = useForm();
+  const [addEvent, { loading, error }] = useMutation(mutation);
+
+  /* date format: yyyy-mm-dd, startTime/endTime: hh:mm.
+  * convert to string yyyy-mm-ddThh:mmZ
+  * parsing with Luxon (fromISO)  to format that supported by all browsers
+  */
+
+  const onSubmit = useCallback(
+    ({ title, description, date, startTime, endTime }) => {
+      addEvent({
+        variables: {
+          title,
+          start: DateTime.fromISO(date + 'T' + startTime + 'Z'),
```

Comment:
We probably don't want to force the user input into UTC, but rather treat the input as local, and set the timezone on the luxon object, so that 17:52+02:00 would become 15:52Z.
```suggestion
          start: DateTime.fromISO(date + 'T' + startTime).setZone('UTC'),
```

Reply:
Done.